Something/WebApplication/ApiControllers/CompaniesController.cs

```diff
@@ -25,21 +25,27 @@ public CompaniesController(UserManager<ApplicationUser> userManager, ICompanySer
         [HttpGet]
         public IEnumerable<CompanyViewModel> Get()
         {
-            var companies = _companyService.GetCompanies();
+            var companies = _companyService.GetCompanies(GetUserId());
             return companies;
         }
         
         // POST: api/Companies
         [HttpPost]
         public async Task<IActionResult> Post([FromBody]string value)
         {
-            if (string.IsNullOrEmpty(value) || value.Length < 3)
+            if (string.IsNullOrWhiteSpace(value) || value.Length < 3)
             {
                 return BadRequest(value);
             }
 
-            var companyId = _companyService.CreateCompanyAsync(value, GetUserId()).ConfigureAwait(false);
-            return Created(string.Empty, companyId);
+            var companyId = _companyService.CreateCompany(value, GetUserId());
+            return Created(string.Format("/api/comapnies/get/{0}", companyId), companyId);
```

Comment:
do not format routes, use Route from ASP.net

Reply:
Example @alirizvi2008 ?